index.html

```diff
@@ -45,36 +56,108 @@
         padding: 20px;
         width: 100%;
       }
+
+      #dice-number,
+      #player-number,
+      #game-mode {
+        width: 60%;
+      }
+
+      #game-mode {
+        font-size: 21px;
+        line-height: 33px;
+        margin: 0 0 23px 0;
+        padding: 0 9px;
+        width: 100%;
+        width: 150px;
+        height: 37px;
+        text-align: center;
+        font-size: medium;
+      }
+
+      #gameRules {
+        width: 100%;
+        text-align: center;
+      }
+
+      #user-input {
+        display: flex;
+        flex-direction: row;
+        flex: auto;
+      }
+
+      .inputInfo {
+        text-align: center;
+        width: 250px;
+      }
+
+      #game-output {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        flex: auto;
+      }
+
+      #output-div {
+        text-align: center;
+      }
     </style>
   </head>
-
   <body>
+
+  </body>
     <h1 id="header">Basics: Beat That! 🚀</h1>
     <div id="container">
-      <p>Input:</p>
-      <input id="input-field" />
+      <div id="gameRules">
+        <p>Hello! Welcome to Beat That! Click <b>start</b> to play the game.</p>
+        <p>
+          Select the number of dice, the number of player, then choose
+          the game mode: </p>
+        <p><b>Beat big</b> or <b>Beat small</b>.</p> 
+        </p>
+        <p>The player with the highest/Lowest number wins! Good luck!</p>
+      </div>
+      <hr />
+      <div id="user-input">
+        <div class="inputInfo">
+          <p>Number of Dice</p>
+          <input
+            type="number"
+            name=""
+            id="dice-number"
+            min="2"
+            value="2"
+            required
+          />
+        </div>
+        <div class="inputInfo">
+          <p>Number of Player</p>
+          <input
+            type="number"
+            name=""
+            id="player-number"
+            min="2"
+            value="2"
+            required
+          />
+        </div>
```

Comment:
i would consider setting a max on the number of dice (and possibly players?)

otherwise i get smth like this which might not be the best game LOL

![image](https://user-images.githubusercontent.com/40411953/156765440-63d6746c-92e1-4d5d-ada2-df7387d8a486.png)